### PR TITLE
[Account] Recent changes to Connect-AZAccount -DeviceCode cause display issues in the PowerShell ISE

### PR DESCRIPTION
**Pull request by lijinpei2008**
## Description
- Fix issue: https://github.com/Azure/azure-powershell/issues/24556

- Group in Azure-PowerShell-Common PR: https://github.com/Azure/azure-powershell-common/pull/426

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [ ] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Accounts/Accounts/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Updated `Connect-AZAccount` to fix a display issue in PowerShell ISE [#24556].
```
Comment (isra-fel) [inline src/Accounts/Accounts/ChangeLog.md:22]:
```suggestion
* Updated `Connect-AzAccount` to fix a display issue in PowerShell ISE [#24556].
```

File: src/Accounts/Accounts/Accounts.csproj [hunk 2]
```diff
@@ -37,6 +37,7 @@
   <ItemGroup>
     <None Update="StartupScripts\AzError.ps1" CopyToOutputDirectory="PreserveNewest" />
     <None Update="StartupScripts\InitializeAssemblyResolver.ps1" CopyToOutputDirectory="PreserveNewest" />
+    <None Update="StartupScripts\InitializePSStyle.ps1" CopyToOutputDirectory="PreserveNewest" />
```
Comment (isra-fel) [inline src/Accounts/Accounts/Accounts.csproj:40]: Have we compared the difference between adding a new script vs. combine it with an existing script?

Comment (lijinpei2008) [reply src/Accounts/Accounts/Accounts.csproj:40]: By way of comparison, adding a new script will be faster at startup and easier to maintain in the future

**Review by BethanyZhou — COMMENTED** [review]
LGTM. Pending approval until common lib has upgraded.

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Pull request merged**